samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/OktaVerify.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2021-Present, Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package info.seleniumcucumber.userStepDefinitions;
+
+import com.okta.sdk.client.Client;
+import com.okta.sdk.client.ClientBuilder;
+import com.okta.sdk.client.Clients;
+import com.okta.sdk.resource.user.User;
+import com.okta.sdk.resource.user.UserBuilder;
+import env.CucumberRoot;
+import env.DriverUtil;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pages.*;
+
+public class OktaVerify extends CucumberRoot {
+
+	protected WebDriver driver = DriverUtil.getDefaultDriver();
+
+	public RegisterPage registerPage = new RegisterPage(driver);
+	public RegisterPhonePage registerPhonePage = new RegisterPhonePage(driver);
+	public SelectAuthenticatorPage selectAuthenticatorPage = new SelectAuthenticatorPage(driver);
+	public VerifyPage verifyPage = new VerifyPage(driver);
+	public OktaVerifyPage oktaVerifyPage = new OktaVerifyPage(driver);
+
+	public ClientBuilder builder = Clients.builder();
+	public Client client = builder.build();
+
+	private final Logger logger = LoggerFactory.getLogger(OktaVerify.class);
+
+
+	@When("^she selects Okta Verify from the list$")
+	public void She_selects_Okta_Verify_from_the_list() {
```

Comment:
Rename all methods to have only lowercase letters. That's the naming convention we have used in other files.

Reply:
Sure, will change it.